manifests/init.pp

```diff
@@ -0,0 +1,67 @@
+# Class: lastpass
+#
+# This class installs the LastPass CLI and provides functions to interact with it.
+
+class lastpass (
+  $package  = $lastpass::params::package,
+  $home     = '/root/.lpass',
+  $agent_timeout = 3600,
+  $username = undef,
+  $password = undef,
+) inherits lastpass::params {
+
+  package { $package:
+    ensure => present,
+  }
+
+  file { "/usr/local/bin/lpasspw":
+    ensure => file,
+    mode   => '0755',
+    source => "puppet:///modules/${module_name}/lpasspw",
+  }
+
+  file { $home:
+    ensure => directory,
+    mode   => '0600',
+  }
+  file { "${home}/pw":
+    ensure  => file,
+    mode    => '0400',
+    content => inline_template("<%= @password %>")
```

Comment:
I don't hink we need inline_template here. This is often used to invoke ruby functions when there is no puppet function to do what you want. I advise against that too...

content takes just text, so I think this should work:
```
content => $password,
```